init.fish

```diff
@@ -10,14 +10,17 @@ if test -z "$FLINE_PATH"
     end
 end
 
-# Use FLINE_PATH to load internals, functions and segments
-if [ "$FLINE_PATH" = "$HOME/.config/fish/conf.d" -a -d "$HOME/.config/fisherman/fishline" ]
-    # Assumes fishline has been installed using fisherman on system without realpath
-    set -gx FLINE_PATH $HOME/.config/fisherman/fishline
-    set -gx fish_function_path $fish_function_path $FLINE_PATH/internals $FLINE_PATH/segments
-else
-    # Standard installation, add function loading path
-    set -gx fish_function_path $fish_function_path $FLINE_PATH/functions $FLINE_PATH/internals $FLINE_PATH/segments
+if test -z "$FLINE_INIT"
+    set -g "$FLINE_INIT" true
```

Comment:
This doesn't work properly on my side, the assignation use the content of the variable to assign and not the variable itself. Also, we should use `set -q` here to check if FLINE_INIT is set as we are the one setting it. We don't need to check that it's empty, just that it exists.

```suggestion
if set -q FLINE_INIT
    set -g FLINE_INIT true
```

Reply:
Good point